src/platforms/react-native/troubleshooting.mdx

```diff
@@ -242,3 +242,19 @@ React Native 0.60.3 tooling was unexpectedly moving source maps files, which cau
 ## Missing Java stacktrace on the New Architecture
 
 If you're using the new architecture on React Native, you might be missing the Java stacktrace. Follow the [issue](https://github.com/facebook/react-native/issues/34923) for more information.
+
+## Failing Build Due to `sentry-cli`
+
+If your build fails due to Sentry CLI error, because you don't have internet connection or [sentry.io](https://sentry.io) is down, you can allow failure of the `sentry-cli` step by adding an environmental variable `SENTRY_ALLOW_FAILURE` equal to `true`. The CLI will still run, but won't cause the build to fail:
```

Comment:
```suggestion
If your build fails due to Sentry CLI error, because you don't have internet connection, or because [sentry.io](https://sentry.io) is down, you can allow failure of the `sentry-cli` step by adding an environmental variable `SENTRY_ALLOW_FAILURE` equal to `true`. The CLI will still run, but won't cause the build to fail:
```